### DISCUSSION TITLE
Distributed version

### DISCUSSION THREAD

**Discussion by buggtb** [microsoft/farmvibes-ai#93] [Q&A] [answered]
With the terraformed deployment, are the plans in the works for an "offically supported" version that runs on AKS, K3D with multiple nodes or wherever in a more distributed mode?

I'm pondering the scale side of this and whilst the single node script offers a good development environment, wider access to a cluster is currently limited.

I guess the converse question to this is, if a distributed version came the projects way, would it get accepted as a PR or is it single node by design?

**Comment by renatolfc** [comment, marked as answer]
Hi @buggtb,

You are correct that there is nothing preventing FarmVibes.AI from running in an actual, multi-node cluster.

We are in the process of testing a set of Azure-based TerraForm scripts that will allow users to setup AKS clusters with FarmVibes.AI.

We are also working on an updated (Python) script for managing the creation and management of said clusters, and should be releasing them in the next few weeks.

**Reply by buggtb** [reply]
Excellent, I'll wait on those then
